client/src/pages/LandingPage/LandingPage.tsx

```diff
@@ -0,0 +1,118 @@
+import CssBaseline from '@material-ui/core/CssBaseline';
+import Box from '@material-ui/core/Box';
+import Grid from '@material-ui/core/Grid';
+import { Formik, FormikHelpers } from 'formik';
+import TextField from '@material-ui/core/TextField';
+import Button from '@material-ui/core/Button';
+import Typography from '@material-ui/core/Typography';
+import useStyles from './useStyles';
+import * as Yup from 'yup';
+import login from '../../helpers/APICalls/login';
+import { useAuth } from '../../context/useAuthContext';
+import { useSnackBar } from '../../context/useSnackbarContext';
+import Image from '/Users/decagon/team-candy-cane/client/src/Images/doglanding.png';
+
+export default function LandingPage(): JSX.Element {
+  const classes = useStyles();
+  const { updateLoginContext } = useAuth();
+  const { updateSnackBarMessage } = useSnackBar();
+
+  const handleSubmit = (
+    { place, drop }: { place: string; drop: string },
+    { setSubmitting }: FormikHelpers<{ place: string; drop: string }>,
+  ) => {
+    login(place, drop).then((data) => {
+      if (data.error) {
+        setSubmitting(false);
+        updateSnackBarMessage(data.error.message);
+      } else if (data.success) {
+        updateLoginContext(data.success);
+      } else {
+        // should not get here from backend but this catch is for an unknown issue
+        console.error({ data });
+
+        setSubmitting(false);
+        updateSnackBarMessage('An unexpected error occurred. Please try again');
+      }
+    });
+  };
+
+  return (
+    <Grid container component="main" className={classes.root}>
+      <CssBaseline />
```

Comment:
The `CssBaseline` is a global reset component and needs to be included only once, ideally in the App.tsx